Gulpfile.js

```diff
@@ -38,14 +41,25 @@ function compile(advanced = false) {
         .pipe(sourcemaps.init())
         .pipe(closureCompiler(options))
         .pipe(sourcemaps.write('/'))
-        .pipe(gulp.dest('dist'));
+        .pipe(gulp.dest('dist/js'));
+}
+
+function cssMinify() {
+    return gulp.src('src/components/**/*.css')
+        .pipe(concatCss('css/erste.css'))
```

Comment:
are you sure? the default version is SIMPLE_OPTIMIZATIONS and the minified version is ADVANCED_OPTIMIZATIONS, although they are not very different. I would say, drop the simple optimizations, and just use erste.js?